doc/crypto/api/ops/aead.rst

```diff
@@ -70,6 +70,10 @@ AEAD algorithms
 
     The CCM block cipher mode is defined in :RFC-title:`3610`.
 
+    .. subsection:: Usage in zigbee
+
+        `PSA_ALG_CCM`, and its truncated variants, can be used to implement CCM* for non-zero tag lengths. CCM* is required by the :cite-title:`ZIGBEE`. For unauthenticated CCM*, the `PSA_ALG_CCM_STAR_NO_TAG` cipher algorithm can be used.
```

Comment:
Please add a hyperlink to the discussion of Zigbee usage under `PSA_ALG_CCM_STAR_NO_TAG`, since that discussion covers both CCM and CCM\*-no-tag.